src/main/scala/au/com/agiledigital/toolform/model/Project.scala

```diff
@@ -14,15 +14,8 @@ import scala.collection.immutable.{IndexedSeq, TreeMap}
   * @param components the components that will be built and combined to form the projects.
   * @param resources  the resources that must be supplied to the project to run.
   * @param topology   the network topology of the environment explains how elements will be related/connected.
-  * @param volumes    the volumes that will be mounted into components.
   */
-final case class Project(id: String,
-                         name: String,
-                         components: Map[String, Component],
-                         resources: Map[String, Resource],
-                         topology: Topology,
-                         volumes: Option[Seq[Volume]],
-                         componentGroups: Option[Seq[ComponentGroup]]) {
+final case class Project(id: String, name: String, components: Map[String, Component], resources: Map[String, Resource], topology: Topology, componentGroups: Option[Seq[ComponentGroup]]) {
```

Comment:
Keep this split over multiple lines for readability.

Reply:
Everytime I do this, scalafmt reverts it to a single line after compiling. How should I proceed?

Reply:
If that's how scalafmt wants it, who am I to complain